shims/hdp26/hbase/pom.xml

```diff
@@ -237,14 +237,15 @@
         <configuration>
           <instructions>
             <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Bundle-Version>${pentaho-osgi-bundles.version}</Bundle-Version>
             <DynamicImport-Package>*</DynamicImport-Package>
             <Import-Package>
               org.pentaho.hadoop.shim.api.*;version=${pentaho-osgi-bundles.version},
               org.pentaho.hadoop.shim.spi.*;version=${pentaho-osgi-bundles.version},
               io.netty.channel;version=${bundle.export.version},
               org.apache.hadoop.conf.*;version=${bundle.export.version}
             </Import-Package>
-            <_exportcontents>org.pentaho.hbase.shim.common,org.apache.hadoop.hbase.*;version=${bundle.export.version}</_exportcontents>
+            <_exportcontents>org.pentaho.hbase.shim.common;version=${bundle.export.version}</_exportcontents>
```

Comment:
Why did we leave out the org.apache.hadoop.hbase.* packages? If we're including hbase jars in this uber jar, we need to export them.

Reply:
None of the other shim's hbase modules export this package. It also does not exist in this project. I removed it for consistency. I doubt it'll break anything.